pkg/ec2provider/ec2provider.go

```diff
@@ -54,10 +55,11 @@ type ec2Requests struct {
 }
 
 type ec2ProviderImpl struct {
-	ec2                ec2iface.EC2API
-	privateDNSCache    ec2PrivateDNSCache
-	ec2Requests        ec2Requests
-	instanceIdsChannel chan string
+	ec2                  ec2iface.EC2API
+	privateDNSCache      ec2PrivateDNSCache
+	ec2Requests          ec2Requests
+	instanceIdsChannel   chan string
+	authenticatorMetrics metrics.Metrics
```

Comment:
We shouldn't need this line anymore?

Reply:
right...its an artifact of bad merge from master..removing it

Reply:
done